vonk/facade/enablechange_1.rst

```diff
@@ -96,16 +98,16 @@ Implementing Create
             return _resourceMapper.MapPatient(_visiContext.Patient.Last());
         }
 
-#.  For the ``Create`` and ``Update`` methods, you will also need to implement the ``NewId`` and ``NewVersion`` methods,
-    because Vonk will call them. For the ``NewId`` method, we will return null, since our ViSi database does not allow us
-    to create our own index value. Since our ViSi repository does not handle versions, we will let the ``NewVersion`` method
-    return null as well::
+*  For the ``Create`` and ``Update`` methods, you will also need to implement the ``NewId`` and ``NewVersion`` methods,
+   because Vonk will call them. For the ``NewId`` method, we will return null, since our ViSi database does not allow us
+   to create our own index value. Since our ViSi repository does not handle versions, we will let the ``NewVersion`` method
+   return null as well::
```

Comment:
Mmm, maybe we should reconsider these functions then.

Reply:
Perhaps, although we still need them to be implementable for other Facades that want to be able to set them.